lib/js/styles/ItemTypography.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div>
+		<!-- @todo fill component -->
+		<div v-for="item in items" :key="item.id" class="itemDefinition">
+			<div v-if="item.label" class="itemDefinition__label">{{ item.label }}</div>
+			<div v-if="item.value" class="itemDefinition__value">{{ item.value }}</div>
+			<div v-if="item.token" class="itemDefinition__token">{{ item.token }}</div>
+			<div v-if="item.attributesRaw" class="itemDefinition__attributes">
+				<div v-for="attribute in item.attributesRaw" :key="item.id + attribute.value">
+					<strong>{{ attribute.property }}</strong
+					>:
+					<span>
+						<span v-pre>{</span>
```

Comment:
can we use `template` tag to limit number of DOM nodes?

Reply:
Yes, fixed.

Reply:
It can't be used, `<template>` need have directive and `v-pre` is not acceptable by lint.

Reply:
I just added a space between brackets:
```
<span>{ {{ attribute.value }} }</span>
```
I think it will be not a problem for UX.